web/index.php

```diff
@@ -175,8 +175,8 @@
 isset($request) || $request = NULL;
 isset($action) || $action = NULL;
 
-if ( ZM_ENABLE_CSRF_MAGIC && $action != 'login' ) {
-    Debug("Calling csrf_check with the following values: \$request = \"$request\", \$view = \"$view\", \$action = \"$action\"");
+if ( ZM_ENABLE_CSRF_MAGIC ) {
```

Comment:
While I do agree our goal should be to remove $action != login, I am concerned about the timing. In order to minimize the pain, perhaps we should first document the old auth method is deprecated, and then remove it in a future version.